storage/grpc_client.go

```diff
@@ -1083,30 +1073,30 @@ func (c *grpcStorageClient) NewMultiRangeDownloader(ctx context.Context, params
 
 	b := bucketResourceName(globalProjectAlias, params.bucket)
 	object := params.object
-	r := &storagepb.BidiReadObjectSpec{
+	bidiObject := &storagepb.BidiReadObjectSpec{
```

Comment:
Maybe call this `readSpec` or `bidiReadSpec`? Since the proto also has object as a data type it's clearer if you indicate that it's a spec (to me anyway).

Reply:
Yes, `readSpec` is good. Was avoiding colliding with the one defined in our own code when I made this change but I renamed that one.